src/main/java/edu/escuelaing/ieti/matchmaking/controllers/LoginController.java

```diff
@@ -0,0 +1,58 @@
+package edu.escuelaing.ieti.matchmaking.controllers;
+
+import edu.escuelaing.ieti.matchmaking.exception.EntityExistsException;
+import edu.escuelaing.ieti.matchmaking.exception.EntityNotFoundException;
+import edu.escuelaing.ieti.matchmaking.model.User;
+import edu.escuelaing.ieti.matchmaking.services.UserService;
+import edu.escuelaing.ieti.matchmaking.util.Token;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.ServletException;
+import java.util.Date;
+
+@RestController
+@RequestMapping("users")
+public class LoginController {
+
+    @Autowired
+    private UserService userService;
+
+    @PostMapping("/login")
+    public ResponseEntity<Token> login(@RequestBody User userLogin) throws ServletException, EntityNotFoundException {
+        String jwtToken = "";
+
+        if (userLogin.getEmail() == null || userLogin.getPassword() == null) {
+            throw new ServletException("Please fill in username and password");
+        }
+
+        String email = userLogin.getEmail();
+        String password = userLogin.getPassword();
+
+        User user = userService.getUserByEmail(email);
+
+        String pwd = user.getPassword();
```

Comment:
This works well, but in a real world application, password are not stored in plain text in the database.